### PR TITLE
Document release-repo scripts

### PR DESCRIPTION
**Pull request by j-rivero**
Over time we have been hosting different scripts for different tasks and we lack of a useful documentation about what is the goal of each of them, requirements, usage and examples. This PR tries to improve the situation.

### CODE REVIEW

File: release-repo-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# -release repositories scripts
+
+The directory hosts the scripts designed to work with Gazebo -release
+repositories, the release repositories hosting the metadata information for
+packaging Debian and Ubuntu. Mostly hosted under
+https://github.com/ignition-release/.
+
+## Scripts available
+
+### bump_dependency.bash
```
Comment (chapulina) [inline release-repo-scripts/README.md:10]: This script doesn't deal just with `-release` repos, so having it here may be misleading. I think it's ok to keep it here though. We could just mention this in the doc?

**Review by chapulina — APPROVED** [review]

File: release-repo-scripts/README.md (hunk 1 above)
Comment (j-rivero) [reply release-repo-scripts/README.md]: Good catch. I create the `source-repo-scripts` to host there the scripts related to use/modify the code in real libraries code in [fddf48f](https://github.com/gazebo-tooling/release-tools/pull/716/commits/fddf48f44317531d209d2f5fae810d156f04ab24). Moved this description to there in [7238e9f](https://github.com/gazebo-tooling/release-tools/pull/716/commits/7238e9f1a2c3712bfffba98850932629003ae9d4)

**Comment by scpeters** [comment]
https://github.com/gazebo-tooling/release-tools/pull/720 adds a new script. I suppose I should preemptively move it to the `source-repo-scripts` folder?

**Comment by j-rivero** [comment]
> #720 adds a new script. I suppose I should preemptively move it to the `source-repo-scripts` folder?

please. Merging this one so you can just rebase on top.

**Pull request merged**